src/core/backbone_router/bbr_manager.hpp

```diff
@@ -117,6 +135,9 @@ class Manager : public InstanceLocator, public Notifier::Receiver
     ThreadStatusTlv::DuaStatus mDuaResponseStatus;
     bool                       mDuaResponseIsSpecified : 1;
 #endif
+
+    MulticastListenersTable mMulticastListenersTable;
```

Comment:
nit: move this above `#if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE` above to keep always-included members variables together?